docs/source/llms/tracing/index.rst

```diff
@@ -227,6 +232,69 @@ for model/API invocations to the active MLflow Experiment.
             :width: 100%
             :align: center
 
+    .. tab:: Swarm
+
+        .. raw:: html
+
+            <h3>OpenAI Swarm Automatic Tracing</h3>
+
+        |
+
+        The MLflow OpenAI flavor supports automatic tracing for **Swarm**, a multi-agent orchestration
```

Comment:
Can we link to the repo? https://github.com/openai/swarm

Reply:
great idea!